backend/btrixcloud/crawl_job.py

```diff
@@ -235,9 +237,18 @@ async def finish_crawl(self):
 
         await self.update_crawl(state=state, finished=self.finished)
 
+        await self.add_crawl_errors_to_mongo()
+
         if completed:
             await self.inc_crawl_complete_stats()
 
+    async def add_crawl_errors_to_mongo(self):
+        """Pull crawl errors from redis and write to mongo"""
+        errors = await self.redis.lrange(f"{self.job_id}:e", 0, -1)
```

Comment:
We might also want to do this incrementally, depending on how many errors we'll have or limit the error list size, eg. if this is pulling down, say a 10000+ list, this may likely crash the redis pod, given current memory limit.

Or, could just store first N entries.
Could do essentially what we're doing here:
https://github.com/webrecorder/browsertrix-crawler/blob/main/util/state.js#L272

Reply:
makes sense!